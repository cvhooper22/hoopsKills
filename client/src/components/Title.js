import React from 'react';
import PropTypes from 'prop-types';

function Title (props) {
  const { title } = props;

  return (
    <h1>
      <i className="fa fa-compass" />
      { ` ${title}` }
    </h1>
  );
}

Title.propTypes = {
  title: PropTypes.string,
};

export default Title;
