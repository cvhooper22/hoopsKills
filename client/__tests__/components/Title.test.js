import React from 'react';
import renderer from 'react-test-renderer';
import Title from '../../src/components/Title';

describe('Title -', () => {

  it('Should render Title', () => {
    const component = renderer.create(
        <Title
          title="This is a test"
        ></Title>
    );

    expect(component).toMatchSnapshot();
  });

});