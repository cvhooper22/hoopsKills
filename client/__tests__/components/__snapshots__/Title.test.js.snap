// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title - Should render Title 1`] = `
<h1>
  <i
    className="ids-icon__dashboard"
  />
   This is a test
</h1>
`;
